src/js/modules/accounts/components/EditProfilePage.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import { connect } from "react-redux";
+import { bindActionCreators } from "redux";
+import { Link } from "react-router-dom";
+
+import { EditUserForm } from "./forms";
+import { updateUser } from "../actions";
+
+const EditProfilePage = ({ session, updateUser }) => {
+  if (!session.user) {
```

Comment:
Why not make that a ternary operator? I'd keep the return statements at the bottom. That way the reader always knows where to look for the actual stuff rendered